.bats/entities.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd $HOME/ardrive-cli/
+
+cache="$(yarn ardrive list-all-drives -w $WALLET)"
+driveID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .driveId' | cut -d '"' -f 2)"
+folderID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .rootFolderId' | cut -d '"' -f 2)"
+
+export PUB_DRIVE_ID="$driveID"
+export PUB_FOLD_ID="$folderID"
```

Comment:
We could replace the above variable declarations for two functions `get_1st_drive_id_from_wallet` and `get_1st_folder_id_from_wallet`. That way we could fancily do the following
```suggestion
export PUB_DRIVE_ID="$(get_1st_drive_id_from_wallet ${WALLET})"
export PUB_FOLD_ID="$(get_1st_folder_id_from_wallet ${WALLET})"
```
But again: it's just to be fancy :nail_care:

Reply:
This is opinionated, gonna keep shorter names for now.